src/main/drivers/accgyro/accgyro.h

```diff
@@ -58,6 +58,9 @@ typedef struct gyroDev_s {
     int32_t gyroZero[XYZ_AXIS_COUNT];
     int32_t gyroADC[XYZ_AXIS_COUNT];                        // gyro data after calibration and alignment
     int16_t gyroADCRaw[XYZ_AXIS_COUNT];
+#if !defined(BEEBRAIN)
```

Comment:
This is not needed - it only impacts RAM, not ROM and for simplicity should be retained.